links.html

```diff
@@ -1,40 +1,46 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
     <meta charset='utf-8'>
-         <title>Carlin's CSD120 Website</title>
-            <link rel="stylesheet" href="styles/main.css">
+    <title>Carlin's CSD120 Website</title>
+    <link rel="stylesheet" href="styles/main.css">
+    <style>
+        main #useful-links a {
+            color:#42B972;
+        }
+    </style>

```

Comment:
missing embedded style (you had the correct rule, but you were using an external stylesheet which is not what the instructions asked for)